.pre-commit-config.yaml

```
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 24.2.0
+    hooks:
+    -   id: black
+        language_version: python3.12
+-   repo: https://github.com/pycqa/isort
+    rev: 5.13.2
+    hooks:
+    -   id: isort
+        args: ["--profile", "black", "--filter-files"] 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)